### PR TITLE
Networking tests fixes

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by EvgenyAgafonchikov** [comment]
United PRs below. Fixed CI build to speedup Networking delivery.
https://github.com/Azure/azure-powershell/pull/6928
https://github.com/Azure/azure-powershell/pull/6935

@akshaysngupta, @tejaskarandikar, @chandrasekarsrinivasan, FYI

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1 [hunk 1]
```diff
@@ -218,23 +218,19 @@ function Test-VirtualNetworkCRUDWithDDoSProtection
         Assert-AreEqual "10.0.1.0/24" $expected.Subnets[0].AddressPrefix
         Assert-AreEqual true $expected.EnableDDoSProtection
         Assert-AreEqual $ddosProtectionPlan.Id $expected.DdosProtectionPlan.Id
-        Assert-AreEqual false $expected.EnableVmProtection
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1]: Why are you removing these asserts?

Comment (chandrasekarsrinivasan) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1]: We have removed this functionality. You can see that we also removed " $expected.EnableVmProtection = $true"

Comment (maddieclayton) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1]: Is this a breaking change in PowerShell?

Comment (chandrasekarsrinivasan) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1]: nope

Comment (maddieclayton) [reply src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1]: How are you removing functionality without making a breaking change? Could users previously use this property and now can't? Can you give a little more background on when/how this was changed? Sorry, want to be sure that we don't break any existing customers.

**Pull request merged**